.circleci/config.yml

```diff
@@ -838,13 +854,8 @@ jobs:
       - attach_workspace:
           at: /tmp/workspace
       - run:
-          name: Load devnet-allocs
```

Comment:
This is necessary - `make devnet-allocs` can take upwards of 5 minutes. The allocs are generated as part of `pnpm-monorepo` and are copied from the workspace into the appropriate directories to avoid rebuilding them. Deleting this increased `op-e2e` runtime from ~4 minutes to ~10.